apps/studio/components/interfaces/SQLEditor/DownloadSnippetModal.tsx

```diff
@@ -2,7 +2,7 @@ import type { ModalProps } from '@ui/components/Modal/Modal'
 import { snakeCase } from 'lodash'
 import Link from 'next/link'
 import { useState } from 'react'
-import { Button, CodeBlock, IconExternalLink, Modal, Tabs, TabsProvider } from 'ui'
```

Comment:
TabsProvider was only necessary to save selection state if there were multiple tabs within the context, and has been superseded by the `withQueryParams` HOC. This doesn't have multiple sets of tabs, so doesn't need either.